code/trace.c

```diff
@@ -1112,16 +1115,21 @@ static Bool traceFindGrey(Seg *segReturn, Rank *rankReturn,
  *
  * This function sets unfixedSummary and fixedSummary such that
  * ScanStateSummary will return the summary passed.  Subsequently fixed
- * references are accumulated into this result.  */
+ * references are accumulated into this result.
+ */
 
 void ScanStateSetSummary(ScanState ss, RefSet summary)
 {
   AVERT(ScanState, ss);
   /* Can't check summary, as it can be anything. */
```

Comment:
This is false now that RefSet is a structured type.  There's no RefSetCheck so `AVERT(RefSet, summary)` isn't possible yet. Investigate.

Reply:
Fixed in e79ee02fe64494f912a7af30b217baa9dae94d5d